playbooks/common/openshift-cluster/metrics/uninstall_metrics.yml

```diff
@@ -0,0 +1,11 @@
+---
+- hosts: oo_first_master
+  tasks:
+  - name: "Remove metrics url from  master config"
+    lineinfile: "state=absent dest=/etc/origin/master/master-config.yaml regexp='^\ \ metricsPublicURL'"
+
+  - name: "Delete metrics objects"
+    command: "{{item}}"
+    with_items:
+      - oc delete all,secrets,templates,sa -l metrics-infra --namespace openshift-infra --ignore-not-found
+        
```

Comment:
This seems to be missing some things, like removal of role bindings:

- [roles/openshift_metrics/tasks/install.yml#L31-L38](https://github.com/openshift/openshift-ansible/blob/bf0f8266e91b552ffa815d3a7bcb65f105606438/roles/openshift_metrics/tasks/install.yml#L31-L38)
- [roles/openshift_metrics/tasks/install.yml#L49-L56](https://github.com/openshift/openshift-ansible/blob/bf0f8266e91b552ffa815d3a7bcb65f105606438/roles/openshift_metrics/tasks/install.yml#L49-L56)